pkgs/by-name/an/angle/package.nix

```
@@ -0,0 +1,159 @@
+{
+  stdenv,
+  gn,
+  ninja,
+  llvmPackages_21,
+  gclient2nix,
+  pkg-config,
+  glib,
+  python3,
+  symlinkJoin,
+  lib,
+  xorg,
+  wayland,
+  pciutils,
+  libGL,
+}:
+let
+  llvmPackages = llvmPackages_21;
+  llvmMajorVersion = lib.versions.major llvmPackages.llvm.version;
+  arch = stdenv.hostPlatform.parsed.cpu.name;
+  triplet = lib.getAttr arch {
+    "x86_64" = "x86_64-unknown-linux-gnu";
+    "aarch64" = "aarch64-unknown-linux-gnu";
+  };
+
+  clang = symlinkJoin {
+    name = "clang";
```

if we're fixing pname, may as-well also add a more detailed name here?

```suggestion
    name = "angle-clang-llvm-join";
```